pkg/config/cli_config.go

```diff
@@ -51,10 +46,6 @@ func (cfg *CLIConfig) Validate() error {
 			cfg.Version, CLIConfigVersions.List()))
 	}
 
-	if cfg.Server == "" {
```

Comment:
This validation makes it so we cannot generate breakglass tokens.

Reply:
Ack